core/src/main/java/org/radargun/config/VmArgs.java

```diff
@@ -139,10 +158,32 @@ private static void ensureArg(Collection<String> args, String arg) {
       @Property(doc = "Min memory", converter = SizeConverter.class)
       private Long min;
 
+      @Property(doc = "Eden memory", converter = SizeConverter.class)
+      private Long eden;
+
+      @Property(doc = "Thread stack size", converter = SizeConverter.class)
+      private Long threadStackSize;
+
+      @Property(doc = "Sets the ratio between young and old generation sizes", converter = NumberConverter.class)
+      private Integer newRatio;
+
+      @Property(doc = "Enables the use of large page memory.")
+      private Boolean useLargePages;
+
       @Override
       public void setArgs(List<String> args) {
-         if (min != null) replace(args, "-Xms", String.valueOf(min));
-         if (max != null) replace(args, "-Xmx", String.valueOf(max));
+         if (min != null)
+            replace(args, "-Xms", String.valueOf(min));
+         if (max != null)
+            replace(args, "-Xmx", String.valueOf(max));
+         if (eden != null)
+            replace(args, "-Xmn", String.valueOf(eden));
+         if (threadStackSize != null)
+            replace(args, "ThreadStackSize", String.valueOf(threadStackSize));
```

Comment:
-Xss is probably more 'portable' flag: http://mail.openjdk.java.net/pipermail/hotspot-dev/2011-June/004272.html